.env.development

```diff
@@ -12,7 +12,6 @@ TERMS_OF_SERVICE_URL=null
 PRIVACY_POLICY_URL=null
 SUPPORT_EMAIL=null
 STUDIO_BASE_URL=http://localhost:18010
-SHOW_ACCESSIBILITY_PAGE=false
```

Comment:
Why did you remove this?

Reply:
I have tried to make it consistent with other links. The accessibility link was redirecting to `<studio_base_url>/accessibility` if SHOW_ACCESSIBILITY_PAGE=true. I have changed it and added a config ACCESSIBILITY_URL. Now, it will display the link if `ACCESSIBILITY_URL` is set and will redirect to the same URL.

Reply:
Should we also look for other usages of this variable in [all other MFEs](https://github.com/search?q=org%3Aopenedx%20SHOW_ACCESSIBILITY_PAGE&type=code) that are using it?

Reply:
Correct.  But this means that this change is a breaking one, so after squashing the commits, the one that remains should be a "feat!:".